pkg/gnubuilder/ldbuilder/ld-builder.go

```diff
@@ -51,15 +51,16 @@ func (ld *LdBuilder) GetName() string {
 }
 
 func (ld *LdBuilder) Analyze(commandline []string) (*service.BuildMessage, error) {
-	if err := ld.parseCommandLine(commandline[1:]); err != nil {
-		return nil, fmt.Errorf("Failed to parse commandline: %v", err)
-	}
-
 	if ld.staticLink {
 		ld.Logger.Printf("ld linking statically")
 	} else {
 		ld.Logger.Printf("ld linking")
 	}
+
```

Comment:
This seems wrong. Isn't the parseCommandLine function the one that finds out if we are statically liniking?